test/Log/LogServiceTest.php

```diff
@@ -58,6 +58,7 @@ public function setUp()
 
         $tempFolder = $this->createTempFolder();
         $this->tempFolder = $tempFolder;
+        $this->createLogFolder();
```

Comment:
I have mixed feelings about  moving this into a separate function. What I don't like is, that it performs "hidden" stuff. It makes the code a little harder to read and doesn't serve any purpose. The function has "3" lines, but if we write this here it could one line. What could be a good reason for this function?

Reply:
Putting the code in a separate function will just label the purpose of the three lines. Without it, I think it might not strike immediately why the three lines are essential while reading the code. Much like `createTempFolder()`. I can document their purpose in the comment.

Reply:
Not convinced. I do like keeping things separate, but this is a border case in which I would opt for simplicity. I was thinking for a moment to use `createTempFolder` within `createLogFolder`. However in the configuration we actually set the workspace path, so that should not be hidden away. 

A lot of these functions should not remain in this class later, because they could be used for other test classes as well. This is just the first time we need them. Down the road we might not just need a log folder, but all the folders in a OPUS workspace. In that case the function definitely makes sense. Maybe add a TODO as reminder to the function comment.

Or course then we have to think about the purpose of `opus4-common`, Does it still fit? Right now I think it still does.

Reply:
Agreed. Will add a TODO comment.